pkg/util/constants.go

```diff
@@ -12,6 +12,8 @@ const (
 	AnnotationVolumeClaimTemplates = prefix + "/volumeClaimTemplates"
 	AnnotationImageID              = prefix + "/imageId"
 	AnnotationHash                 = prefix + "/hash"
+	AnnotationVolumeStatus         = prefix + "/volume-status"
+	AnnotationDetachedVM           = prefix + "/detached-vm"
```

Comment:
It is simply following the existing ones in constants.go, people needs to check the implementation part to understand what those constants are doing.

I don't mind adding comments here, but it will looks unusual comparing existing code.

Reply:
IMO it's worth doing even only these two annotations containing comments. There are lots of code have no comment explaining their business logic and for me it's hard to figure them out. Still, only my own opinion. The decision is up to you 😃.

Reply:
Updated.